src/Gui/InputHintWidget.cpp

```diff
@@ -0,0 +1,319 @@
+        case KeyClear: return tr("Clear");
+
+        // cursor movement
+        case KeyHome: return tr("Home");
+        case KeyEnd: return tr("End");
+        case KeyLeft: return QStringLiteral("←");
+        case KeyUp: return QStringLiteral("↑");
+        case KeyRight: return QStringLiteral("→");
+        case KeyDown: return QStringLiteral("↓");
+        case KeyPageUp: return tr("PgDown");
+        case KeyPageDown: return tr("PgUp");
+
+        // modifiers
+#ifdef FC_OS_MACOSX
+        case KeyShift: return QStringLiteral("⇧");
+        case KeyControl: return QStringLiteral("⌘");
+        case KeyMeta: return QStringLiteral("⌃");
+        case KeyAlt: return QStringLiteral("⌥");
+#else
+        case KeyShift: return tr("Shift");
+        case KeyControl: return tr("Ctrl");
+#ifdef FC_OS_WIN32
+        case KeyMeta: return tr("⊞ Win");
+#else
+        case KeyMeta: return tr("❖ Meta");
+#endif
+        case KeyAlt: return tr("Alt");
+#endif
+        case KeyCapsLock: return tr("Caps Lock");
+        case KeyNumLock: return tr("Num Lock");
+        case KeyScrollLock: return tr("Scroll Lock");
+
+        // function
+        case KeyF1: return QStringLiteral("F1");
+        case KeyF2: return QStringLiteral("F2");
+        case KeyF3: return QStringLiteral("F3");
+        case KeyF4: return QStringLiteral("F4");
+        case KeyF5: return QStringLiteral("F5");
+        case KeyF6: return QStringLiteral("F6");
+        case KeyF7: return QStringLiteral("F7");
+        case KeyF8: return QStringLiteral("F8");
+        case KeyF9: return QStringLiteral("F9");
+        case KeyF10: return QStringLiteral("F10");
+        case KeyF11: return QStringLiteral("F11");
+        case KeyF12: return QStringLiteral("F12");
+        case KeyF13: return QStringLiteral("F13");
+        case KeyF14: return QStringLiteral("F14");
+        case KeyF15: return QStringLiteral("F15");
+        case KeyF16: return QStringLiteral("F16");
+        case KeyF17: return QStringLiteral("F17");
+        case KeyF18: return QStringLiteral("F18");
+        case KeyF19: return QStringLiteral("F19");
+        case KeyF20: return QStringLiteral("F20");
+        case KeyF21: return QStringLiteral("F21");
+        case KeyF22: return QStringLiteral("F22");
+        case KeyF23: return QStringLiteral("F23");
+        case KeyF24: return QStringLiteral("F24");
+        case KeyF25: return QStringLiteral("F25");
+        case KeyF26: return QStringLiteral("F26");
+        case KeyF27: return QStringLiteral("F27");
+        case KeyF28: return QStringLiteral("F28");
+        case KeyF29: return QStringLiteral("F29");
+        case KeyF30: return QStringLiteral("F30");
+        case KeyF31: return QStringLiteral("F31");
+        case KeyF32: return QStringLiteral("F32");
+        case KeyF33: return QStringLiteral("F33");
+        case KeyF34: return QStringLiteral("F34");
+        case KeyF35: return QStringLiteral("F35");
+
+        default: return tr("???");
```

Comment:
Why is this marked for translation?

Reply:
Because it might not be the preferred way to symbolize unknowns in all cultures.